sublime-package.json

```diff
@@ -46,12 +46,17 @@
                     "intelephense.compatibility.correctForBaseClassStaticUnionTypes": {
                       "type": "boolean",
                       "default": true,
-                      "markdownDescription": "Resolves `BaseClass|static` union types to `static` instead of `BaseClass`."
+                      "description": "Resolves `BaseClass|static` union types to `static` instead of `BaseClass`.",
```

Comment:
Why changing from `markdownDescription` to `description` and loosing markdown formatting?

Reply:
Copy of intelephense, I will fix it

Reply:
> Copy of intelephense, I will fix it

may file a pr to intelephense too